apps/careeropportunity/migrations/0014_auto_20191031_1239.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.1.11 on 2019-10-31 11:39
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('careeropportunity', '0013_careeropportunity_website'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='careeropportunity',
+            name='website',
+        ),
+        migrations.AddField(
+            model_name='careeropportunity',
+            name='applicationLink',
+            field=models.URLField(blank=True, null=True, verbose_name='søknadslenke'),
```

Comment:
This migration I am quite sure you need to change, because as far as I know,  we still want to keep the current data.
Perhaps [migrations.RenameField](https://docs.djangoproject.com/en/2.2/ref/migration-operations/#renamefield) does what we want

Reply:
I have looked into it and I disagree:

1. The field "website" is not the same as "application link". They don't mean the same thing, and the information in website would not work as an application link.

2. Since the website field is brand new there is almost no career opportunities that have this set. And in all cases where it is set, the link is to the homepage of the company and would not work as an application link.